docs/data_types/tensor/tensor.md

```diff
@@ -0,0 +1,125 @@
+# Tensor
+
+
+
+DocArray support several tensor types that can be used inside BaseDoc. 
+
+The main ones are:
+
+- [`NdArray`][docarray.typing.tensor.NdArray] for numpy tensors
+- [`TorchTensor`][docarray.typing.tensor.TorchTensor] for Pytorch tensors
+- [`TensorFlowTensor`][docarray.typing.tensor.TensorFlowTensor] for tensorflow tensors
+
+The three of them wrap from their respective framework tensor type. 
+
+!!! note
+    [`NdArray`][docarray.typing.tensor.NdArray] and [`TorchTensor`][docarray.typing.tensor.TorchTensor] are a subclass of their native tensor type. This means that they can be used natively inside their framework.
+
+!!! note
+    [`TensorFlowTensor`][docarray.typing.tensor.TensorFlowTensor] stores the pure `tf.Tensor` object inside the `tensor` attribute. This is due to a limitation on the TensorFlow framework that does not allow to subclass the `tf.Tensor` object.
+
+DocArray also supports [`AnyTensor`][docarray.typing.tensor.AnyTensor] which is the Union of the three previous tensor types. 
+This is a generic placeholder to specify that it can work with any tensor type (numpy, torch, tensorflow).
```

Comment:
```suggestion
DocArray also supports [`AnyTensor`][docarray.typing.tensor.AnyTensor], which is the Union of the three previous tensor types. 
This is a generic placeholder to specify that it can work with any tensor type (NumPy, PyTorch, TensorFlow).
```